clusters/app.ci/ci-cluster-pool/ci-ocp-4.7.0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4.7.0-amd64-aws-us-east-1
+  namespace: ci-cluster-pool
+  labels:
+    product: ocp
+    version: 4.7.0
+    architecture: amd64
+    cloud: aws
+    owner: dpp
+    region: us-east-1
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  imageSetRef:
+    name: ocp-4.7.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: jenkins-ci-iam-aws-creds
```

Comment:
We likely need this to be in it's own account, as otherwise it will impact our ability to manage quota with boskos

Reply:
Not sure if you have seen `maxSize` (on top of `size`) in the cluster pool
https://github.com/openshift/hive/blob/73585aafdf3e43be77dc3eac29c3f27bede2a153/apis/hive/v1/clusterpool_types.go#L28

But yes, it will impact the quota as well.
So I should start to request a new account from DPP?